rest_api/config_apis/infrastructure-config-openshift-io-v1.adoc

```diff
@@ -490,7 +490,7 @@ Type::
 
 | `nodeDNSIP`
 | `string`
-| nodeDNSIP is the IP address for the internal DNS used by the nodes. Unlike the one managed by the DNS operator, `NodeDNSIP` provides name resolution for the nodes themselves. There is no DNS-as-a-service for BareMetal deployments. In order to minimize necessary changes to the datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames to the nodes in the cluster.
+| nodeDNSIP is the IP address for the internal DNS used by the nodes. Unlike the one managed by the DNS operator, `NodeDNSIP` provides name resolution for the nodes themselves. There is no DNS-as-a-service for BareMetal deployments. To minimize necessary changes to the datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames to the nodes in the cluster.
```

Comment:
I'm pretty sure we don't want to manually update files within the rest_api/ folder, because the API docs are auto-generated each release. Is my understanding correct here @jboxman?

Same applies for all files below that are in the rest_api/ folder.

Reply:
This is news to me, so I'm glad you pointed it out. I'll revert all of those changes.